tortoise/verifying_tortoise_test.go

```diff
@@ -1060,13 +985,6 @@ func TestBaseBlock(t *testing.T) {
 	alg.HandleIncomingLayer(context.TODO(), l3.Index())
 	require.Equal(t, int(types.GetEffectiveGenesis().Add(1).Uint32()), int(alg.LatestComplete().Uint32()))
 	expectBaseBlockLayer(l2.Index(), 0, defaultTestLayerSize, 0)
-
-	// mark all blocks bad
-	alg.trtl.GoodBlocksIndex = make(map[types.BlockID]bool, 0)
-	baseBlockID, exceptions, err := alg.BaseBlock(context.TODO())
-	r.Equal(errNoBaseBlockFound, err)
```

Comment:
i removed this assertion because we are selecting bad block if good is not found, will probably add a test with empty state to check this, otherwise we will always select something